chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.13.1"
-version: 0.10.2
+version: 0.10.3
```

Comment:
```suggestion
version: 0.10.2
```

We bump versions before making the chart release, to ensure we only bump it once per release.

Reply:
oook